SimPEG/EM/FDEM/RxFDEM.py

```diff
@@ -11,17 +12,27 @@ class BaseRx(SimPEG.Survey.BaseRx):
     """
 
     def __init__(self, locs, orientation=None, component=None):
-        assert(orientation in ['x','y','z']), "Orientation {0!s} not known. Orientation must be in 'x', 'y', 'z'. Arbitrary orientations have not yet been implemented.".format(orientation)
-        assert(component in ['real', 'imag']), "'component' must be 'real' or 'imag', not {0!s}".format(component)
+        assert(
+            orientation in ['x', 'y', 'z']
+        ), "Orientation {0!s} not known. Orientation must be in 'x', 'y', 'z'."
+        " Arbitrary orientations have not yet been implemented.".format(
+            orientation
+        )
+        assert(
+            component in ['real', 'imag']
```

Comment:
I am still not sure if 'component' is the right word. I had a hard time explaining this - generally when referring to 'component' the first association is with the spatial component. eg. "we are measuring the x-component of the electric field". Perhaps `part in ["real", "imag"]` ?